libs/core/src/components.d.ts

```diff
@@ -233,39 +233,39 @@ export namespace Components {
          */
         "componentId": string;
         /**
-          * Indicates whether or not the input field is disabled
+          * If true, the input field is disabled.
          */
         "disabled"?: boolean;
         /**
-          * Specifies the error message and provides an error-themed treatment to the field
+          * Specifies the error message and provides an error-themed treatment to the field.
          */
         "errorMessage"?: string;
         /**
-          * Displays a message or hint below the input field
+          * Displays a message or hint below the input field.
          */
         "helperMessage"?: string;
         /**
-          * Indicates whether or not the input field is invalid or throws an error
+          * If true, the input field is invalid.
```

Comment:
What does it mean for the input field to be invalid? Does this change the appearance? Can the user interact with it? Do we know if it triggers a message? 

Can we be more descriptive here so that it's clear the treatment when the input field is invalid? Like, "If true, the input field no longer accepts alien emojis"